README.md

```diff
@@ -92,6 +92,21 @@ redis = { version = "0.27.5", features = ["tokio-rustls-comp"] }
 redis = { version = "0.27.5", features = ["async-std-rustls-comp"] }
 ```
 
+Add `rustls` to dependencies
+
+```
+rustls = { version = "0.23", features = ["ring"] }
+```
+
+And then, early in the main fn, add:
```

Comment:
"And then, before creating a connection, ensure that you install a crypto provider. For example:

Reply:
hi @nihohit, I updated the sentence.